### DISCUSSION TITLE
Use custom build with Vercel deployments

### DISCUSSION THREAD

**Discussion by yarinsa** [vercel/community#2276] [Help]
### Summary

Vercel deployments are more then great! 
But I have to configure a github integration with it. Meaning that I have to wait for Vercel to "listen" to my commit, then create a deployment and then response to this deployment in pretty hacky way and run the E2E tests and etc against the deployed url.
What I would want to achieve is to build the docker/static files in my CI then use 'vercel deploy'. And get back a URL. 
This way I will be able to get all the features from vercel such as "creating deployment anf github, comments" and etc. But have a customized pipeline .

**Comment by amyegan** [comment]
Hi @yarinsa! You can disable the GitHub integration with [`git.deploymentEnabled: false`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled) and use the Vercel CLI for custom workflows: https://vercel.com/guides/using-vercel-cli-for-custom-workflows

**Reply by yarinsa** [reply]
But what if I want to "enjoy" the deployments,domain, comments and all the others features I am getting by the integration? Do I need to have them on my own?
